redwood-treehouse/src/commonMain/kotlin/app/cash/redwood/treehouse/FlowWithInitialValue.kt

```diff
@@ -1,32 +0,0 @@
-/*
- * Copyright (C) 2022 Square, Inc.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package app.cash.redwood.treehouse
-
-import kotlin.native.ObjCName
-import kotlinx.coroutines.flow.Flow
-import kotlinx.coroutines.flow.StateFlow
-import kotlinx.serialization.Contextual
-import kotlinx.serialization.Serializable
-
-@Serializable
-@ObjCName("FlowWithInitialValue", exact = true)
-public data class FlowWithInitialValue<T>(
```

Comment:
This class causes an internal compiler issue in `1.8.20`. Fortunately the next Zipline release will include the `StateFlow` support so we can delete this as part of the upgrade.